src/components/SlateEditor/plugins/rephrase/Rephrase.tsx

```diff
@@ -82,7 +82,11 @@ export const Rephrase = ({ attributes, editor, element, children }: Props) => {
       })
       .then((res) => setGeneratedText(res))
       .catch((err: NdlaErrorPayload) =>
-        toast.error({ title: t("textGeneration.failed.variant"), description: err.messages, removeDelay: 1500 }),
+        createMessage({
```

Comment:
Dette vil gi en dialog inni en dialog, vet ikke om det er dumt?

Reply:
@Jonas-C noen ideèr?

Reply:
Jeg stemmer for å bare vise feilmeldingen inni dialogen nederst som jeg skrev over 😅

Reply:
Synes det er fint å differensiere mellom faktiske permanente feil (for mye innhold i felt/ugyldig verdi) og midlertidige feil. Kanskje dialog i dialog er greiest